app/models/user.rb

```diff
@@ -0,0 +1,18 @@
+class User < ApplicationRecord
+
+  validates :first_name, :last_name,  presence: true, length: { maximum: 60 }
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
```

Comment:
add more specs for email, in your specs you check only if email is present, write specs for this regex